sgl-kernel/csrc/cpu/qkv_proj.cpp

```diff
@@ -624,3 +629,69 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> qkv_proj_with_rope(
 
   return std::make_tuple(q_input, k_input, v_input);
 }
+
+std::tuple<at::Tensor, at::Tensor, at::Tensor> fused_qkv_proj_with_rope(
```

Comment:
not a very good naming ...
```suggestion
std::tuple<at::Tensor, at::Tensor, at::Tensor> qkv_proj_with_rope_fused_weight(
```